app/javascript/components/public_health/PatientsTable.js

```diff
@@ -728,6 +761,16 @@ class PatientsTable extends React.Component {
               assigned_users={this.state.assigned_users}
             />
           )}
+          {this.state.action === 'Flag for Follow-up' && (
+            <FollowUpFlag
+              patients={this.state.table.rowData.filter((_, index) => this.state.selectedPatients.includes(index))}
```

Comment:
I'm probably being a bit OCD here, but can we make the ordering of the props consistent with all the other bulk edit components?